test/src/main/java/org/bf2/test/TestUtils.java

```diff
@@ -165,16 +166,29 @@ public static Function<InputStream, InputStream> replacer(final Map<String, Stri
      * Restart kubeapi
      */
     public static void restartKubeApi() {
```

Comment:
I'd be fine with removing this method and the test that is based upon it.  This will not cause an informer to relist, just the underlying watches to reconnect.  We also no longer have our own informer logic.